src/@rocketseat/gatsby-theme-docs/components/Logo.js

```diff
@@ -1,13 +1,32 @@
 import React, { useContext } from "react";
 import styled from "@emotion/styled";
-import Toggle from "../../../components/Toggle";
 import { NightModeContext } from "../../../providers/NightModeProvider";
+import Moon from "../../../components/Icons/Moon";
+import Sun from "../../../components/Icons/Sun";
 import logo from "../assets/izm.png";
 
 const Wrapper = styled.div`

```

Comment:
Please make this wrapper 100% width, because it looks a bit off on the preview

Reply:
Done!